docker/README.md

```diff
@@ -1,20 +1,22 @@
 # Introduction
 
-This README describes two ways to deploy the containers for web UI and the workers.
+This README describes two ways to deploy the containers for the openQA web UI
+and the workers.
 
 The first one describes how to deploy an openQA environment using docker with
 the Fedora images or images built locally.
 
-The second one uses a docker-compose to deploy a complete web UI with HA for the
+The second one uses docker-compose to deploy a complete web UI with HA for the
 API and the dashboard.
 
-If a section or action is tagged as "Fedora", it shall be applied only for
-Fedora deployment. On the contrary, if the tag "docker-compose" is used it will
-be used for the docker-compose deployment.
+If a section or action is tagged as "Fedora", it shall be applied only for a
```

Comment:
```suggestion
If a section or action is tagged as "Fedora", it will be applied only for a
```

Reply:
That's not right. nothing "will be applied", the user has to do it.

Reply:
*shall* in the archaic sense means, it will happen. Perhaps what you meant was "should"?

Reply:
*I* did not mean anything, it was there in before. I have exchanged the word "shall" with "should" now to avoid the ambiguity.